src/pybind/mgr/volumes/fs/fs_util.py

```diff
@@ -75,6 +75,27 @@ def listdir(fs, dirpath):
         raise VolumeException(-e.args[0], e.args[1])
     return dirs
 
+def listsnaps(fs, snapdirpath, filter_inherited_snaps=False):
+    """
+    Get the snap names from a given .snap directory path
+    """
+    if os.path.basename(snapdirpath) != b".snap":
```

Comment:
snapshot directory base name '.snap' should not be hard coded? It is mgr.rados.conf_get('client_snapdir') or volspec.snapshot_prefix

Reply:
Done